flask_lesson_advanced/blueprint/template_supermarket/all_supermarkets.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% block title %}All Supermarkets{% endblock %}
+{% block body %}
+<h1>Supermarkets</h1>
+    <ol>
+        {% for i in value %}
+        <li>
+            {% if i['name'] in sess %}
+                <a href="{{url_for('supermarket.supermarkets_fun')}}/{{ i['id'] }}" onclick="return false;">{{ i['name'] }}</a>
```

Comment:
`url_for` has parameter for `/{{ i['id'] }}`

Reply:
we had chat about it and it no need to be changed